model-optimizer/mo/utils/check_ie_bindings.py

```diff
@@ -33,8 +33,8 @@ def send_telemetry(mo_version: str, message: str, event_type: str):
 
 def import_core_modules(silent: bool, path_to_module: str):
     try:
-        from openvino.inference_engine import IECore, get_version # pylint: disable=import-error
-        from openvino.offline_transformations import ApplyMOCTransformations, CheckAPI # pylint: disable=import-error
+        from openvino.inference_engine import get_version, read_network # pylint: disable=import-error
+        from openvino.offline_transformations import ApplyMOCTransformations, ApplyLowLatencyTransformation, GenerateMappingFile # pylint: disable=import-error
```

Comment:
The same question here. Do we really need all these imports?

Also, we don't need `import re` in line 7.

Reply:
Comment was added.